AWS_CloudWatch/FIleMetricPublisher.py

```diff
@@ -0,0 +1,79 @@
+from Functions.MetricPublisher import MetricPublisher
+from Functions.Metrics import CounterMetric
+import unittest
+from datetime import datetime
+import json
+import os
+import time
+
+
+class FileMetricPublisher(MetricPublisher):
+    def __init__(self,file_location:str,name_space:str):
+        self.file_location = file_location
+        self.name_space = name_space
+        self.counter_metric = CounterMetric(name_space=self.name_space)
+
+    def inc_key_count(self, metric_name: str):
+        data = self.counter_metric.publish_metric(metric_name=metric_name)
```

Comment:
this seems way more complicated than it should be - we should have a file with the value in it

something like <namespace>/<metric-name>.json

the operation to increment should simply read the value and write the new value (old value + 1)

What am I missing?